maps/BlackthornCity.asm

```diff
@@ -13,14 +13,14 @@ BlackthornCity_MapScripts:
 	def_scene_scripts
 
 	def_callbacks
-	callback MAPCALLBACK_NEWMAP, .FlyPoint
-	callback MAPCALLBACK_OBJECTS, .Santos
+	callback MAPCALLBACK_NEWMAP, BlackthornCityFlyPointCallback
```

Comment:
Let's standardize on "`Flypoint`", not "`FlyPoint`". (Currently a lot of the callbacks use "`.FlyPoint`", but that's counter to the `Flypoints` table label, `wStartFlypoint`, `wEndFlypoint`, and the fact that we use `FLYPOINT` not `FLY_POINT`.)